### ISSUE TITLE
Partial backpropagation

### ISSUE THREAD

**Issue by dhananjaisharma10** [open-mmlab/mmdetection#1181]
Hi!

What would be the best way to train a model only partially? For instance, I want to train a retinanet but keeping the backbone weights untouched.

Edit: I noticed this parameter `frozen_stages`. If I set it to 4, in the case of a resnet50, I should not be getting any gradients across the backbone, right?

Please let me know.

Thanks!

**Comment by hellock** [comment]
Yes you are right. BTW, you can also manually frozen the parameters in the model, just like what it done in the backbone.
